src/features/acf/diabetes/common/model.ts

```diff
@@ -39,3 +50,21 @@ export type DiabetesAcfItem = {
 };
 
 export type AcfDashboardType = "HIPERTENSAO" | "DIABETES";
+
+export const sortableFields = [
+    "latestExamRequestDate",
+    "mostRecentAppointmentDate",
+    "hemoglobinTestDueDate",
+    "nextAppointmentDueDate",
+    "conditionIdentifiedBy",
+    "patientCpfOrBirthday",
+    "patientName",
+    "patientAge",
+    "visitantCommunityHealthWorker",
+] as const;
+
+export type SortableFields = (typeof sortableFields)[number];
```

Comment:
```suggestion
export type SortableField = (typeof sortableFields)[number];
```

Apesar da lista ser de vários `sortableFields`, esse tipo representa apenas 1 deles, então ele deveria estar no singular. A mesma coisa se aplica a todos os tipos desse arquivo que são gerados a partir de uma lista, como esse.